app-text/poppler/poppler-0.50.0.recipe

```diff
@@ -0,0 +1,298 @@
+SUMMARY="A PDF rendering library"
+DESCRIPTION="Poppler is a PDF rendering library based on the xpdf-3.0 code \
+base."
+HOMEPAGE="https://poppler.freedesktop.org/"
+COPYRIGHT="2002-2008 Albert Astals Cid
+	2005 Kristian Høgsberg
+	2001-2002 Derek Noonburg (xpdf)"
+LICENSE="GNU GPL v2"
+REVISION="1"
+DATAVERSION=0.4.7
+SOURCE_URI="https://poppler.freedesktop.org/poppler-$portVersion.tar.xz"
+SOURCE_URI_2="https://poppler.freedesktop.org/poppler-data-$DATAVERSION.tar.gz"
+
+CHECKSUM_SHA256="c9c93318b789d3933f6e0bad3bc65110280c28eac3f0666284bb9c9a0ab4bc36"
+CHECKSUM_SHA256_2="e752b0d88a7aba54574152143e7bf76436a7ef51977c55d6bd9a48dccde3a7de"
+
+SOURCE_DIR="poppler-$portVersion"
+SOURCE_DIR_2="poppler-data-$DATAVERSION"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+ARCHITECTURES_data="any"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	$portName = $portVersion
```

Comment:
I guess using "$portName" in PROVIDES works fine but I think it would be better to have:
```
	poppler$secondaryArchSuffix = $portVersion
```
instead of
```
	$portName = $portVersion
```